release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -127,6 +127,13 @@ For more information, see xref:../networking/hardware_networks/about-sriov.adoc#
 You can update the Ingress Controller to switch between an AWS Classic Load Balancer (CLB) and an AWS Network Load Balancer (NLB) without deleting the Ingress Controller.
 
 For more information, see xref:../networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws.adoc[Configuring ingress cluster traffic on AWS].
+[id="ocp-4-12-ipv6-gratuitous-ARPs-default-SR-IOV"]
+
+==== IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on the SR-IOV CNI plugin
+
+Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plugin (where the IP address management CNI plug-in has assigned IPs) now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/Neighbor caches with the correct information.
```

Comment:
@ShaunaDiaz, we generally try to avoid parenthetical fragments per IBM style guide. Can we offset this with commas?

We've been hyphenating _plug-in_ as well.

I might lowercase _neighbor_ as well; I don't think the neighbor cache is generally uppercase.